.gitignore

```diff
@@ -20,3 +20,4 @@ doc/build/
 .pdm.toml
 __pypackages__
 .pdm-python
+pdm.py
```

Comment:
Why is this needed?

Reply:
was an extra file in my repo when i went to commit, not sure where it came from